swiiny/create-nextjs-dapp
# Chore/bump packages

## Description

This pull request bumps various packages, including MUI, base packages, styled components, and tailwind. Additionally, several fixes have been applied, including adjustments to MUI button margins and a mobile logo sizing issue.

## Type of Change


## Steps to Test

1. Pull the latest changes.
2. Ensure all dependencies are properly installed.
3. Test various components and utilities affected by the package updates.
4. Verify MUI button margins display correctly.
5. Confirm the placeholder logo size on mobile devices.

## Expected Behavior

All packages should be updated to their latest versions without introducing any breaking changes. MUI button margins should appear correctly, and the placeholder logo on mobile devices should be displayed at an appropriate size.

## Additional Information

- Bumped MUI, base packages, styled components, and tailwind packages.
- Fixed MUI button margin top issue.
- Adjusted the placeholder logo size for mobile view.
